terraform/deploy-to-gcr/modules/mydrive-gcr-s3/variables.enc.tf

```diff
@@ -0,0 +1,21 @@
+{
+	"data": "ENC[AES256_GCM,data:R/dRZsn6tm3cYPHshykA1/uQ2WcDWfG/Zvfu2+Fj7RQIIlK9rjd45xZ7VPfjJms9OXqj3Y2GXzOzVfxluWSiiIOca6mmLNtUbtATk4zJRAeY5S2Y8YShh82KyvZsdJyWCRYCC4+V2SCJlK82I5bdXu8D7NButL88dPnwCvdX+Riq9qLY9NiH2zznosnqgRyDS7TUj0xmVq30M9/IfW5SAi5kdNEtB2eX0e/fG1mS0yl35/k6moYVsImBxJhk/t2W09M2tOl3L23/tyrcC7/lBmLTWk60QcnIa7EvvOGrBuFba15IZXRm5eZ1vkx5ZBV50C+cI/eb9483rT+mjeCEbfhyprBNUKJ4EKmSEDLBZBoSXBb22t0NU9AQWGgwQeidTdUTEZFzk3bRPmOJZmDwWWeJjvyBFefxOk9rTnNNOlLpF25vgPIhKAn+O5Lka4YJHis2+819D0iOu1SUEi8TSTrIQOoLGzHeedRly9OfMsndXDppejapM3YCGhZqwEwkrTt8MrtC5yFzC8TGOw62G0O2Cgjo6bhdQZ3BF/p+7cD2LFtkYVQtWhf8NDbnfk33yXsdMimervTiAkVVieXvGDAWsEskS16/HaUjcal2Biut0eTgzrX1XZVyhcuZpM1yI6Nx5DMdCMUotcuhmlHtdC/i4wJmpfUUKM4ZYVpYLC90e0wuYixTKDdwZLPkDjU2lY+Jzl0MfZ0wYkKZbMoKiFRjTxGwijkoL30a6XXzvX0RHWcCvzlZfwlFsdXpkeRfpwGBPU7v1nzU8U4YG9o+1k98vgdvZ81K/eRTZhJ50mJFzJ50agpUu33lEa5Ridk6gcGZwmtt9g4xn0MKEfCCghLSux0lNGtjZie7hzpvUFikNzMB+KnlIP1uU1lWYVbX1ekL+gRw4TDr5QUyEUxeeW1YundTyFRdpzLQFkV5vfk0d59b0LUwQtqvPtOtJ3KsXaxuF8f34SIY9822a95AuUrD+ZGzRv6bRitDcKmiHl0bKlKZ4lkeCuMhKxI7PpZAKKvaI67KnG5uw2FRe0Ls7Pz5vhUkUBw/bON1uCpr6fggwM9MmcuY1TwFaIV8ekgW7bg2uGYxoKg7Dko5zUeCkAunfJl8I4DMfbzHwCXpx7sH2KtNEQ6saG4UsBIptECnX0xOanc9LFnD/rbBasSFqt2qba80LseJIdv6tY1eHD9FRgmkfKwZ5j1Zcpblb54SSoln0hrEOgimbjYmbwGMAgHAMro+OiM6RGl5zYkMqc6Rl1APe5OJ9AvFvTNj496/1HksvNqqxHNySuDsC9mvuzm8vwvexO7bdueU0xxWirmYu14x8ZuwUzS0CHru7+zyeMOIpyeSS79RnIquc6HFrbhaSFhwhXHdoswSwdzx5c32vikmQK6w6D7KvT0XVK5aJC/5hHuSF6g5u2/xCFNqnod5nvxsheD2M4lEv9ti88jpj+6Fq6Km+dJGQ1EPoRN3Oz32X3tE7qz9ztwy2dOIKxk+ZEZSrygXn6/ViRYcoXU9/bScj4VH+exH87pVdvqfFWpsGeFKtnAXgJAadmqIVKcFfK3Oe8FV/cZ9,iv:OkUydlEPXXNi5vuJzmD91/tfroqAawsTiIPg9Kh3wh4=,tag:HVWpFBHdz5eGPLpOT1YFJw==,type:str]",
```

Comment:
would be better to store and encrypt vars as json

Reply:
need to discuss this